dateparser/custom_language_detection/langdetect.py

```diff
@@ -0,0 +1,36 @@
+from langdetect.detector_factory import DetectorFactory, PROFILES_DIRECTORY
+import langdetect
+
+
+# The below Factory is set to prevent setting global state of the library
+# but still get consistent results.
+# Refer - https://github.com/Mimino666/langdetect
+
+class Factory:
+    data = None
+
+
+def init_factory():
+    if Factory.data is None:
+        Factory.data = DetectorFactory()
+        Factory.data.load_profile(PROFILES_DIRECTORY)
+        Factory.data.seed = 0
+
+
+def detect_langs(text):
```

Comment:
To me name `detect_langs` is confusing with another method `detect_lagnuages`

Based on what it returns do you think a better name can be `get_language_probablities`

Reply:
Thanks, @kishan3, I feel the same so I have changed it to _get_language_probablities

Reply:
Done 👍🏻